js/functions.js

```diff
@@ -0,0 +1,52 @@
+// Функция для проверки длинны строки
+const currentString = 'Какой-то текст';
+const minLenght = 10;
+function stringMath() {
+  return currentString.length >= minLenght;
+}
+stringMath();
```

Comment:
Проверяемые значения лучше подставить в качестве параметра функции
```suggestion
// Функция для проверки длинны строки
const currentString = 'Какой-то текст';
const minLenght = 10;
function stringMath(currentString, minLength) {
  return currentString.length >= minLength;
}
stringMath('Какой-то текст', minLength);
```